### ISSUE TITLE
rejectedWith is missing a return type annotation

### ISSUE THREAD

**Issue by mykter** [shouldjs/should.js#156]
Hi,

Typescript complains about these two definitions:
```
should.d.ts(172,5): error TS7010: 'rejectedWith', which lacks return-type annotation, implicitly has an 'any' return type.
should.d.ts(173,5): error TS7010: 'rejectedWith', which lacks return-type annotation, implicitly has an 'any' return type.
```

Should.js version 13.1.0

**Comment by btd** [comment]
Yep, thanks. Really miss it.

**Comment by btd** [comment]
Fixed in 13.1.1
